Formula/luvit.rb

```diff
@@ -32,13 +33,14 @@ class Luvit < Formula
   # https://github.com/luvit/lit/raw/$(LIT_VERSION)/get-lit.sh
   resource "luvi" do
     url "https://github.com/luvit/luvi.git",
-        tag:      "v2.12.0",
-        revision: "5d1052f11e813ff9edc3ec75b5282b3e6cb0f3bf"
+        tag:      "v2.14.0", # Install a newer version to fix ARM64.
+        revision: "450ec116aef6f8cb0cba83c7e61c9bce112d6deb"
   end
```

Comment:
If this works, can we make a PR upstream for it?

Reply:
Do you just mean to update the `luvi` version in `get-lit.sh` (which is what we reference for the `luvi` version)?

Reply:
Yeah, if it works for us it could be worth consideration for upstream